### PR TITLE
Have a working query language by merging models

### PR DESCRIPTION
**Pull request by gcandal**
Move what was decoupled in `utils` to inside the model, similarly to https://github.com/hivesolutions/appier/blob/master/src/appier/model.py .

### CODE REVIEW

**Comment by gcandal** [comment]
@joamag this one's for you but can't assign 😞

**Comment by joamag** [comment]
> @joamag this one's for you but can't assign 😞

lol :)

File: js/util/data.js [hunk 1]
```diff
@@ -54,13 +54,8 @@ export const SORT_MAP = {
...
 };
 
-export const getDataObject = function(
-    params,
-    alias = true,
-    page = true,
-    find = true,
-    norm = true
-) {
+export const getDataObject = function(params = {}, options = {}) {
```
Comment (joamag) [inline js/util/data.js]: rename to getObject

File: js/util/data.js [hunk 2]
```diff
@@ -125,8 +120,13 @@ const _findTypes = function(params) {
     return result;
 };
 
-const _findDefaults = function(params) {
+const _findDefaults = function(params, options) {
```
Comment (joamag) [inline js/util/data.js]: default options to empty object

**Pull request merged**